src/main/java/teammates/ui/webapi/action/ActionFactory.java

```diff
@@ -63,7 +63,6 @@
         map(ResourceURIs.RESPONSE_COMMENT, PUT, UpdateFeedbackResponseCommentAction.class);
         map(ResourceURIs.RESPONSE_COMMENT, DELETE, DeleteFeedbackResponseCommentAction.class);
         map(ResourceURIs.RESULT, GET, GetSessionResultsAction.class);
-        map(ResourceURIs.RESULT_CSV, GET, GetSessionResultsAsCsvAction.class);
```

Comment:
Why is this removed?

Reply:
Here are my investigations after trying to bring back the function:

I think we only have two options here because of the couplings:

**Keep the second commit**:
- Need to rewrite the CSV generation in the backend.
- Or write a new function in the frontend for CSV generation.
 
Personally, I think the CSV generation should be in the frontend where we care about the data presentation (Actually the current refactor is targeted to make this happened in the frontend to minimized the code duplication (e.g., missing response generation)). 

I have done some initial attempts and I would say it requires the same efforts to do either. After all, the CSV view is just a non-interactive version of the per question view in the frontend.

**Remove the second commit**:
- This will always work as commits are independent.
- We will remove other functions later. Not sure when though.

Personally, I would like to be ambitious and do it in frontend with the interns :P